### DISCUSSION TITLE
[Application Development] How to enable "eth_subscribe" interface?

### DISCUSSION THREAD

**Discussion by maodaishan** [ethereum-optimism/developers#144] [Application Development]
### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

 I tried "eth_subscribe" interface in my private deployed OP Stack rollup, it replyed "notifications not supported", Is there anyway I can enable this interface? thanks.

### Additional Information

Here's the interactions:
Request:
curl -X POST -H "Content-Type: application/json" --data '{ "id": 1, "jsonrpc": "2.0", "method": "eth_subscribe", "params": ["newHeads"] }' http://xx.xx.xx.xx:8545

Response:
{"jsonrpc":"2.0","id":1,"error":{"code":-32601,"message":"notifications not supported"}}

**Comment by mslipper** [comment]
You can't use eth_subscribe on an `http` endpoint - you have to use web sockets. Add `--ws --ws.api=eth` to fix.

**Comment by maodaishan** [comment]
Thanks @mslipper , I could use it now  :)
